prow/plugins/lifecycle/close.go

```diff
@@ -30,7 +30,7 @@ import (
 var closeRe = regexp.MustCompile(`(?mi)^/close\s*$`)
 
 func init() {
-	plugins.RegisterGenericCommentHandler("close", deprecatedCloseHandleComment, closeHelp)
+	plugins.RegisterGenericCommentHandler("close", handleCloseGenericComment, closeHelp)
```

Comment:
The effect of this is to actually un-deprecate the deprecated `close` plugin.

Since it is past April 2018, the intent here is to delete the `close` plugin (so delete this line entirely) and require people who want close functionality to enable the `lifecycle` plugin instead (note that our primary prow deployment no longer uses close, just lifecycle: https://github.com/kubernetes/test-infra/blob/a0c58f0342abd64286ebf71bbdb579f71eceb4c4/prow/plugins.yaml#L343

This might be better in its own PR (or its own commit)

Reply:
Let me know whether you want to delete the close plugin in this PR or handle it separately? If separately can you add back the deprecated stuff?

Reply:
> Let me know whether you want to delete the close plugin in this PR or handle it separately?

I split this into two commits in this PR.

1. The first commit removes the deprecated stuff and uses the lifecycle plugin for `/close` and `/reopen`.
2. The second commit makes sure that only a single lifecycle label is persisted.